app/controllers/sessions_controller.rb

```diff
@@ -8,8 +8,13 @@ def create
     user = User.find_by(email: params[:email])
 
     if user && user.authenticate(params[:password])
-      session[:user_id] = user.id
-      redirect_to home_path, notice: 'You have signed in.'
+      if user.active?
+        session[:user_id] = user.id
+        redirect_to home_path, notice: 'You have signed in.'
+      else
+        flash[:danger] = 'Your account has been suspended.'
```

Comment:
You have flash danger and error. Feels confusing a little bit to mix both styles.